content_scripts/github/username.js

```diff
@@ -84,6 +84,8 @@ function initializeGitHubIdQueries() {
             hrefException: true,
         }
     );
+    // projects (beta): splice by assignees
+    _addQuery(`projects-v2 div[data-testid^="slicer-panel"] div > ul > li span h3"`, { hrefException: true });
```

Comment:
Suggesting to make the query a bit more specific: match only h3s next to a user avatar image.

Otherwise we would also match with "No assignees" or any other value if the view is sliced by another field. Bit tricky with the selectors because the avatar is a child in a different subtree in the DOM, but `:has()` pseudo-class works here.

```suggestion
    // projects (beta): slice by assignees
    _addQuery(
        `projects-v2 div[data-testid="slicer-panel"] li div.actionlistitem-leadingcontent:has(div > img[data-testid="github-avatar"]) + div > div > h3`,
        {
            hrefException: true,
        }
    );
```